ortb/request_validator_test.go

```diff
@@ -140,6 +154,12 @@ func TestValidateImpExt(t *testing.T) {
 					expectedImpExt: `{"prebid":{"bidder":{"appnexus":{"placement_id":555}}}}`,
 					expectedErrs:   []error{},
 				},
+				{
+					description:    "Valid bidder root ext + Empty Prebid Bidder",
```

Comment:
Please add a comment to explain the valid bidder is ignored because prebid.bidder is present.